### PR TITLE
[keek] fix info extraction

### PR DESCRIPTION
**Pull request by remitamine**

### CODE REVIEW

**Comment by dstftw** [comment]
Uploader still can be extracted.

**Comment by remitamine** [comment]
i added code to extract description, uploader and uploader id.

**Comment by dstftw** [comment]
```
Traceback (most recent call last):
  File "youtube_dl/__main__.py", line 16, in <module>
    import youtube_dl
  File "C:\Dev\git\youtube-dl\master\youtube_dl\__init__.py", line 43, in <module>
    from .extractor import gen_extractors, list_extractors
  File "C:\Dev\git\youtube-dl\master\youtube_dl\extractor\__init__.py", line 272, in <module>
    from .keek import KeekIE
  File "C:\Dev\git\youtube-dl\master\youtube_dl\extractor\keek.py", line 17
SyntaxError: Non-ASCII character '\xc3' in file C:\Dev\git\youtube-dl\master\youtube_dl\extractor\keek.py on line 17, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
```

**Comment by remitamine** [comment]
it works now in both python 2 and python 3

File: youtube_dl/extractor/keek.py [hunk 1]
```diff
@@ -1,46 +1,42 @@
...
-        'md5': '09c5c109067536c1cec8bac8c21fea05',
+        'url': 'https://www.keek.com/keek/NODfbab',
+        'md5': '9b0636f8c0f7614afa4ea5e4c6e57e83',
         'info_dict': {
             'id': 'NODfbab',
             'ext': 'mp4',
-            'uploader': 'youtube-dl project',
-            'uploader_id': 'ytdl',
-            'title': 'test chars: "\'/\\\u00e4<>This is a test video for youtube-dl.For more information, contact phihag@phihag.de .',
+            'title': 'test chars: "\'/\\\xe4<>This is a test video for youtube-dl.For more information, contact phihag@phihag.de . - Video - Videos on Keek',
```
Comment (dstftw) [inline youtube_dl/extractor/keek.py]: You should add a coding cookie on top of the file instead.

Comment (remitamine) [inline youtube_dl/extractor/keek.py]: > You should add a coding cookie on top of the file instead.

you mean?
```python
# coding: utf-8
```

Comment (dstftw) [inline youtube_dl/extractor/keek.py]: Yes.

**Comment by remitamine** [comment]
i added the utf-8 coding cookie

File: youtube_dl/extractor/keek.py [hunk 2]
```diff
@@ -1,46 +1,43 @@
...
-        if raw_desc:
-            uploader = self._html_search_regex(
-                r'Watch (.*?)\s+\(', raw_desc, 'uploader', fatal=False)
-            uploader_id = self._html_search_regex(
-                r'Watch .*?\(@(.+?)\)', raw_desc, 'uploader_id', fatal=False)
-        else:
-            uploader = None
-            uploader_id = None
+        uploader = uploader_id = None
+        matches = re.search(r'data-username="(?P<uploader>[^"]+)"[^>]*data-user-id="(?P<uploader_id>[^"]+)"', webpage)
```
Comment (dstftw) [inline youtube_dl/extractor/keek.py]: Why not just use `self._search.regex` twice?

Comment (remitamine) [inline youtube_dl/extractor/keek.py]: because they are extracted from the same tag.

Comment (dstftw) [inline youtube_dl/extractor/keek.py]: This does not prevent from using `self._search_regex` in any way.

Comment (remitamine) [inline youtube_dl/extractor/keek.py]: i will change to use self._search_regex.

**Comment by remitamine** [comment]
i changed it to use _search_regex.

**Comment by dstftw** [comment]
Cherry-picked, thanks.

**Pull request closed without merging**